code/game/objects/effects/decals/posters/bs12.dm

```diff
@@ -293,3 +293,40 @@
 	icon_state="bsposter60"
 	name = "John Fleet poster"
 	desc = "A poster produced by the SCGF. It depicts Petty Officer John Fleet, a character used to boost troop morale, standing in front of the SCG flag proudly. Nobody knows if he really existed."
+
+///TeguStation posters
+
+/// Moth poster sprites are created by Azlanon and AyyRobotics
+/decl/poster/tegu_1
+	icon_state ="tposter1"
+	name = "Safety Moth - Poisoning"
+	desc = "This informational poster uses Safety Moth(TM) to ask the viewer not to poison the station donuts."
+
+/decl/poster/tegu_2
+	icon_state ="tposter2"
+	name = "Safety Moth - Hardhats"
+	desc = "This informational poster uses Safety Moth(TM) to tell the viewer to wear hardhats at all times. It's like a lamp for your head!"
+/decl/poster/tegu_3
+	icon_state ="tposter3"
+	name = "Safety Moth - Smokey?"
+	desc = "Only YOU can prevent plasma fires."
```

Comment:
Seems a bit odd that this is the only one that doesn't say "This information poster uses Safety Moth(TM)."

Reply:
> Seems a bit odd that this is the only one that doesn't say "This information poster uses Safety Moth(TM)."

as for this one thats what the original poster says and I like it more

Reply:
Plasma is also called phoron on baycode, so make sure you change that as well.